tests/operator/suite.go

```diff
@@ -120,15 +116,52 @@ func LoadChecks() {
 			return nil
 		}))
 
-	checksGroup.Add(checksdb.NewCheck(identifiers.GetTestIDAndLabels(identifiers.TestOperatorCatalogSourceBundleCountIdentifier)).
-		WithSkipCheckFn(testhelper.GetNoCatalogSourcesSkipFn(&env)).
+	checksGroup.Add(checksdb.NewCheck(identifiers.GetTestIDAndLabels(identifiers.TestOperatorInstallationInTenantNamespace)).
+		WithSkipCheckFn(testhelper.GetNoOperatorsSkipFn(&env)).
 		WithCheckFn(func(c *checksdb.Check) error {
-			testOperatorCatalogSourceBundleCount(c, &env)
+			testOperatorInstallationInTenantNamespace(c, &env)
 			return nil
 		}))
 }
 
-// This function check if the Operator CRD version follows K8s versioning
+/*
+Checks :
+
+	Operators with SingleNamespaced install mode should only be installed in the tenant dedicated operator namespace
+*/
+func testOperatorInstallationInTenantNamespace(check *checksdb.Check, env *provider.TestEnvironment) {
+	check.LogInfo("Starting testInstalledSingleNamespaceOperatorInTenanttNamespace")
+
+	var compliantObjects []*testhelper.ReportObject
+	var nonCompliantObjects []*testhelper.ReportObject
+
+	singleNamespacedOperators := filterSingleNamespacedOperatorUnderTest(env.Operators)
+
+	for _, operator := range singleNamespacedOperators {
+		operatorNamespace := operator.Namespace
+
+		check.LogInfo("Checking operator %s in namespace %s ", operator.Name, operator.Namespace)
+
+		doesNamespaceContainsOnlyOperatorPods, err := checkIfNamespaceContainsOnlyOperatorPods(operatorNamespace)
+
+		if err != nil {
+			check.LogError("Skipped - cannot proceed with operator %s", operator.Name)
+			continue
+		}
+		if doesNamespaceContainsOnlyOperatorPods {
+			check.LogInfo("Operator %s has valid installation in tenant namespace %s ", operator.Name, operatorNamespace)
+			compliantObjects = append(compliantObjects, testhelper.NewOperatorReportObject(operator.Namespace, operator.Name,
+				"Operator has valid installation in tenant namespace ", true).AddField(testhelper.OperatorName, operator.Name))
```

Comment:
The operator's name is lready set in the NewOperatorReportObject() call. Is that `AddField()` with the name field actually needed?